src/Components/Components/src/Rendering/Renderer.Log.cs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Components.Rendering
+{
+    public abstract partial class Renderer
+    {
+        internal static class Log
+        {
+            private static readonly Action<ILogger, int, Type, int, Type, Exception> _initializingChildComponent =
+                LoggerMessage.Define<int, Type, int, Type>(LogLevel.Trace, new EventId(1, "InitializingChildComponent"), "Initializing component {ComponentId} ({ComponentType}) as child of {ParentComponentId} ({ParentComponentId})");
```

Comment:
These all should be `LogLevel.Debug`. In practice `Trace` means debugging message that contains **sensitive** data like a user's credit card number. None of these have sensitive data.

Reply:
note: we also don't consider things likes URLs to be sensitive, because it would be hard to log anything comprehensible for users without including them.

Reply:
OK. The docs described a difference based on level of detail which sounded pretty subjective, so I took a guess based on how deep in the system they are.

Happy to switch them to `Debug` if this is the convention.

Reply:
I'm sure the docs say something. In practice we use debug unless it's sensitive data, then we use trace.